### ISSUE TITLE
Research: Add tariff zones to stop place details view UI

### ISSUE THREAD

**Issue by Leitsi** [HSLdevcom/jore4#1727]
This is a research ticket. Create implementation ticket once the technical specification is ready. Note, there are already frontend notions at the end of this ticket, once the new implementation ticket is created you can copy those to the new ticket.

* Research Jore3 way to do this, ask Juha-Matti or Petri Selkisalo, Markku Huotari for further details. 
* Research Tiamat options for this, ask Teemu for furhter details.
* Specify & implement to Jore4

Workshop: Juha-Matti, (optional: Petri Selkisalo - contact in slack if needed), Markku Huotari, Teemu

Keep a meeting and specify how this should be done. After, implement.

**Notes for the future:**

Tariff zones can be fetched from open HSL data https://public-transport-hslhrt.opendata.arcgis.com/datasets/hsln-maksuvy%C3%B6hykkeet/explore?location=60.233368%2C24.767591%2C8.52

See schema proposal in https://dev.azure.com/hslfi/JORE%204.0/_wiki/wikis/JORE-4.0.wiki/3561/Stop-location#

- [x] investigate how to add tariff zones to Tiamat
- [ ] add tariff zones to seed data
- [ ] display tariff zones in stop place UI

Added & combiner from other ticket:

It should be possible to add tariff zones to stops, as this is needed information for ticket systems which get their data from Jore4.

Design proposal: [Tariff zone is included in the stop register data (Figma).](https://www.figma.com/file/dsNMixsO1558hCX0XzRDrc/Bus-stops-%7C-JORE-4.0-UX?type=design&node-id=1118%3A61159&mode=design&t=XFU31jLoUBDAqmOX-1)

![Image](https://github.com/HSLdevcom/jore4/assets/77675592/b2b686bb-5228-4895-86cc-26cf1b9099c7)

Possibly related to https://github.com/HSLdevcom/jore4/issues/1728
